belgif-rest-problem/src/main/java/io/github/belgif/rest/problem/InvalidAccessTokenProblem.java

```diff
@@ -47,9 +51,43 @@ public class InvalidAccessTokenProblem extends ClientProblem {
 
     private static final long serialVersionUID = 1L;
 
+    private String reason = "The access token is invalid";
```

Comment:
Field can be made final

Reply:
Only if we make some changes to the initialization / constructor chaining logic.

How do you see this? The default constructor calling `this("The access token is invalid");`?

Reply:
I was thinking about:

```
    private final String reason;

    public InvalidAccessTokenProblem() {
        this(DETAIL, "The access token is invalid");
    }

    public InvalidAccessTokenProblem(String reason) {
        this(DETAIL + ": " + reason, reason);
    }

    private InvalidAccessTokenProblem(String detail, String reason) {
        super(TYPE_URI, HREF, TITLE, STATUS);
        setDetail(detail);
        this.reason = reason;
    }
```